mix.exs

```diff
@@ -85,4 +86,20 @@ defmodule Skate.MixProject do
       {:ueberauth, "~> 0.9.0"}
     ]
   end
+
+  # Aliases are shortcuts or tasks specific to the current project.
+  # For example, to install project dependencies and perform other setup tasks, run:
+  #
+  #     $ mix setup
+  #
+  # See the documentation for `Mix` for more info on aliases.
+  defp aliases do
+    [
+      setup: ["deps.get", "ecto.setup", "assets.setup"],
+      "ecto.setup": ["ecto.create", "ecto.migrate"],
+      "ecto.reset": ["ecto.drop", "ecto.setup"],
+      "assets.setup": ["cmd npm --prefix=assets install"],
```

Comment:
**suggestion (non-blocking):** I don't have a strong feeling on it, but it may be better to use `npm ci` since I think that's more likely to produce an end result that exactly matches your `package-lock.json` without changing anything.

Reply:
Interesting, I just timed it on my old 2018 MacBook and `npm ci` for Skate took only 17 seconds. I wonder what's different - if you have an M1 maybe there are binaries that need to be compiled that come prepackaged for x86 or something like that?

In any event, either way I like the `assets.reset` idea!

Reply:
> Just clarifying, with `assets.reset`, are you good with leaving `assets.setup` as is?

Yeah that sounds good to me. 👍

Reply:
Totally random, but it was only yesterday that I learned that `npm ci` stood for "clean install" as opposed to "continuous integration," although a continuous integration environment is one of the contexts where you might want to use the clean install command.